_docs/LitmusVagrantTesting.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Testing Puppet Modules with Litmus and Vagrant
+---
+
+## Testing Puppet Modules with Litmus and Vagrant
+ 
+### The Setup 
+
+For this demo I'm gonna run through the entire process from start to finish, I hope that you find it useful.
+To start with the first thing that you need to do is clone down the module that you want to test, in this case we'll be using `puppetlabs-scheduled_task`. This can easily be accomplished with a simple command:
+
+```
+git clone https://github.com/user-name/puppetlabs-scheduled_task.git
+```
+
+Now that you have the module cloned down simply run the below command to set up the environment:
+
+```
+bundle install --path .bundle/gems/
+```
+
+### Provisioning the Machine
+
+Now that the environment is setup it is time to provision the machine, while normally you would do this with a vagrant command, Litmus can do it for you if provided with the correct information, as shown below:
+
+![Litmus Vagrant Testing - provisioning the machine](Vagrant%20Provision.png)
+
+If you are testing against multiple different versions then you can also utilise the provision_list command, which let's you set out an entire suite of OS to test against by adding them to and then calling them from the `provision.yml`.
```

Comment:
```suggestion
If you are testing against multiple different versions then you can also utilise the `provision_list` command, which adds/appends an entire suite of OSs to the `provision.yml`.
```

Reply:
Changed to: 
If you are testing against multiple different OSs then you can also utilise the `provision_list` command, which alows you to provision an entire suite of machines from pre-set lists found within the `provision.yml` file.